python/tests/test_append_only_fs_bucket.py

```diff
@@ -90,3 +90,17 @@ def wait_and_lock_second():
         # Verify that both threads were able to acquire the lock
         self.assertFalse(lock_acquired[0], "The first thread should have released the lock")
         self.assertTrue(lock_acquired[1], "The second thread should have acquired the lock after the first thread released it")
+
+    def test_get_size(self):
+        bucket = AppendOnlyFSBucket(self.mem_base_bucket, self.locks_path)
+        object_name = "test_object"
+        content = b"test content"
+
+        bucket.put_object(object_name, content)
+
+        size = bucket.get_size(object_name)
+        self.assertEqual(size, len(content))
+
+        with self.assertRaises(FileNotFoundError):
+            bucket.get_size(f"non_existent_object_{threading.get_ident()}_{time.time_ns()}")
```

Comment:
This is non-deterministic behavior, which is forbidden in unit-tests, and in this context, these functional tests are covering the role of the unit-tests.
We don't want flacky tests, so let's keep deterministic behavior in this test battery